sky/clouds/service_catalog/config.py

```diff
@@ -8,22 +8,26 @@
 
 
 @contextlib.contextmanager
-def _set_use_default_catalog(value: bool):
-    old_value = get_use_default_catalog()
+def _set_use_default_catalog_if_failed(value: bool):
+    old_value = get_use_default_catalog_if_failed()
     _thread_local_config.use_default_catalog = value
     try:
         yield
     finally:
         _thread_local_config.use_default_catalog = old_value
 
 
-# Whether the caller requires the catalog to be narrowed down
-# to the account-specific catalog (e.g., removing regions not
-# enabled for the current account) or just the raw catalog
-# fetched from SkyPilot catalog service. The former is used
-# for launching clusters, while the latter for commands like
-# `show-gpus`.
-def get_use_default_catalog() -> bool:
+def get_use_default_catalog_if_failed() -> bool:
+    """Whether to use default catalog if failed to fetch account-specific one.
+
+    Whether the caller requires the catalog to be narrowed down to the account-
+    specific catalog (e.g., removing regions not enabled for the current account
+    or use region name for the AWS account).
+
+    When set to True, the caller allows to use the default service catalog,
+    which may have staled information, e.g. region names, but it is ok for the
```

Comment:
```suggestion
    which may have stale information, e.g. region names, but it is ok for the
```
Also, do we mean zone names or region names?

Reply:
Ahh, it should be the zone name. Updated. Thanks!